src/searchContext.js

```diff
@@ -0,0 +1,37 @@
+"use strict";
+
+var utils = require("../utils");
+
+module.exports = function(defaultFuncs, api, ctx) {
+  return function searchContext(messageID, threadID, isGroup, limit, direction, callback) {
+    if(!callback && utils.getType(threadID) === 'Function') {
+      return callback({error: "Pass a threadID as a second argument."});
+    }
+    if (!callback) {
+      throw {error: "searchContext: need callback"};
+    }
+
+    var tmpForm = {
+      client: 'web_messenger',
+      message_id: messageID,
+      limit: limit || 6,
+      direction: direction,
+    };
+    if (isGroup) {
```

Comment:
Could you remove `isGroup` and use the same trick we used in `sendMessage` https://github.com/Schmavery/facebook-chat-api/blob/9da530343e2510388bc8cb481f1074d67c463df1/src/sendMessage.js#L155
We make a call to `getUserInfo` and do `Object.keys(res).length > 0`, if it's `true` then it's a 1-1 thread, if it's false then it's a group thread.